docs/1.x/configuration.md

```diff
@@ -28,18 +28,28 @@ relay.maxmemory = 0
+| `relay.max_endpoint_dbs`          | `32`             | The maximum number of PHP workers that will have their own in-memory cache. While each PHP worker will have its own connection to Redis, not all workers need their own in-memory cache and can be read-only workers. This setting is per connection endpoint (distinct Redis connections), e.g. connecting to two separate instances will double the workers. |
+| `relay.initial_readers`           | `128`            | The number of epoch readers allocated on startup. |
+| `relay.invalidation_poll_freq`    | `5`              | How often (in microseconds) Relay should proactively check the connection for invalidation messages from Redis. |
+| `relay.loglevel`                  | `off`            | Whether Relay should log debug information. Supported levels: `debug`, `verbose`, `error`, `off` |
+| `relay.logfile`                   |                  | The path to the file in which information should be logged, if logging is enabled. |
+| `relay.cluster.seeds`             |                  | The list of cluster nodes addresses grouped by cluster name, which will be used to initialize each cluster, encoded as URL query string, e.g. `cluster1[]=127.0.0.1:7000&cluster2[]=127.0.0.1:8000` |
+| `relay.cluster.auth`              |                  | The list of credentials for each cluster, encoded as URL query string. Password string or username/password pairs may be used, e.g. `cluster1=secret&cluster2[]=username&cluster2[]=secret` |
+| `relay.cluster.timeout`           |                  | The maximum number of seconds Relay will wait while establishing connection to a single cluster node. |
+| `relay.cluster.read_timeout`      |                  | The maximum number of seconds Relay will wait while reading from a cluster node. |
+| `relay.session.locking_enabled`   | `0`              | Whether to enable session locking to avoid race conditions and keep session data consistent across requests. |
+| `relay.session.lock_expire`       | `0`              | The number of seconds while Relay will try to acquire lock. When value is zero or negative `max_execution_time` will be used. |
+| `relay.session.lock_retries`      | `0`              | The number of attempts Relay will try to acquire lock. If value is zero or negative `100` will be used to be compatible with PhpRedis. |
+| `relay.session.lock_wait_time`    | `0`              | The number of microseconds Relay will wait between each attempt to acquire lock. If value is zero or negative `20000` will be used to be compatible with PhpRedis. |
+| `relay.session.compression`       | `none`           | Compression algorithm used for session data. Supported values: `lzf`, `lz4`, `zstd` and `none` |
+| `relay.session.compression_level` |                  | The used compression level. |
```

Comment:
Is there no default value for `relay.session.compression_level` or `relay.cluster.*timeout`

Reply:
For `compression_level` empty value means that default level will be used at the discretion of the compressor
Empty cluster timeouts act similar to not set timeouts in constructor, Relay will decide default value